src/torchmetrics/detection/mean_ap.py

```diff
@@ -373,7 +375,7 @@ class MeanAveragePrecision(Metric):
     def __init__(
         self,
         box_format: Literal["xyxy", "xywh", "cxcywh"] = "xyxy",
-        iou_type: Union[Literal["bbox", "segm"], tuple[str]] = "bbox",
+        iou_type: Union[Literal["bbox", "segm", "keypoints"], tuple[str]] = "bbox",
```

Comment:
```suggestion
        iou_type: Union[Literal["bbox", "segm", "keypoints"], tuple[str, str]] = "bbox",
```
just fixing the tuple type which seems to be wrong